apps/main/lib/main/entities/article.rb

```diff
@@ -0,0 +1,20 @@
+require "redcarpet"
+require "rom/struct"
+
+module Main
+  module Entities
+    class Article < ROM::Struct
+      def body_html
```

Comment:
@AMHOL @halogenandtoast Yep, I was considering that a view part would the ideal final place for this, but I was conscious about not introducing too many things all at once in this PR.

That said, if we moved it to a view part, it means we could _remove_ this entity for now, so that's not a bad trade-off. I'll go ahead and do that :)